server/athenian/api/internal/miners/github/pull_request.py

```diff
@@ -2903,27 +2903,29 @@ def __call__(self, pr: MinedPullRequest) -> PullRequestFacts:
         )
         if closed and first_review_exact and first_review_exact > closed:
             first_review_exact = None
-        first_comment_on_first_review = first_review_exact or merged
-        if first_comment_on_first_review:
+        potential_first_comment_on_first_review = first_review_exact or merged
+        if potential_first_comment_on_first_review:
             committed_dates = pr.commits[PullRequestCommit.committed_date.name]
             try:
                 last_commit_before_first_review = committed_dates[
-                    committed_dates <= first_comment_on_first_review
+                    committed_dates <= potential_first_comment_on_first_review
                 ].max()
             except ValueError:
                 last_commit_before_first_review = None
             if not (last_commit_before_first_review_own := bool(last_commit_before_first_review)):
-                last_commit_before_first_review = first_comment_on_first_review
+                last_commit_before_first_review = potential_first_comment_on_first_review
```

Comment:
@vmarkovtsev last_commit_before_first_review seems not used. Can it be removed also from facts ?